server/database/config/connection.ts

```diff
@@ -0,0 +1,51 @@
+/* eslint-disable no-console */
+import { Sequelize, DataTypes } from 'sequelize';
+
+const {
+  NODE_ENV, DB_URL, TEST_DB_URL, DATABASE_URL,
+} = process.env;
+
+let dbUrl:string = '';
+let sslConnection: boolean | object = false;
+
+if (NODE_ENV === 'test') {
+  dbUrl = TEST_DB_URL || '';
+  sslConnection = false;
+} else if (NODE_ENV === 'production') {
+  dbUrl = DATABASE_URL || '';
+  sslConnection = { rejectUnauthorized: true };
+} else if (NODE_ENV === 'dev') {
+  dbUrl = DB_URL || '';
+  sslConnection = false;
+} else {
+  console.log('no environment found');
+}
+
+const sequelize = new Sequelize(dbUrl, {
+  dialectOptions: { sslConnection },
+});
+
+sequelize
```

Comment:
Use `async` `await` instead of  `.then()`
try to see this [article](https://github.com/FarahZaqout/intro-async-await)